__tests__/app.test.js

```diff
@@ -186,3 +186,50 @@ describe("GET: /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  let comment;
+  beforeEach(() => {
+    comment = {
+      username: "lurker",
+      body: "This is a comment",
+    };
+  });
+
+  it("201: respond with the posted comment", () => {
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(comment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body).toMatchObject({
+          comment_id: expect.any(Number),
+          body: "This is a comment",
+          votes: expect.any(Number),
+          author: "lurker",
+          article_id: 1,
+          created_at: expect.any(String),
+        });
+      });
+  });
+
+  it("404: respond with an error for posting on an ID that does not exist", () => {
+    return request(app)
+      .post("/api/articles/1000/comments")
+      .send(comment)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Article not found");
+      });
+  });
+
+  it("400: respond with an error for posting on an ID that is not possible", () => {
+    return request(app)
+      .post("/api/articles/banana/comments")
+      .send(comment)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid ID");
+      });
+  });
```

Comment:
A couple more tests for this one! 

201 - ignores unnecessary properties in a sent comment
400 - missing body or username
404 - username does not exist